server.js

```diff
@@ -3,36 +3,71 @@ import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
+import booksData from './data/books.json'
 
 const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
-const port = process.env.PORT || 8080
+const Book = mongoose.model('Book', {
+  bookID: Number,
+  title: String,
+  authors: String,
+  average_rating: Number,
+  isbn: Number,
+  isbn13: Number,
+  language_code: String,
+  num_pages: Number,
+  ratings_count: Number,
+  text_reviews_count: Number
+})
+
+if (process.env.RESET_DB) {
+  const seedDatabase = async () => {
+    await Book.deleteMany()
+
+    booksData.forEach((item) => {
+      new Book(item).save()
+    }) 
+  }
+  seedDatabase()
+}
+
+const port = process.env.PORT || 8098
 const app = express()
 
-// Add middlewares to enable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
-
-// Start defining your routes here
+ 
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Welcome!')
+})
+
+app.get('/books', async ( req, res) => {
+  const allBooks = await Book.find()
+  res.json(allBooks)
+})
+
+app.get('/books/book/:bookID', async (req, res) => {
+  const oneBook = await Book.findOne( { bookID: req.params.bookID })
+  if (oneBook) {
+    res.json(oneBook)
+  } else {
+    res.status(404).json({ error: 'No book found with that ID' })
+  }
+})
+
+app.get('/books/authors/:authorName', async (req, res) => {
```

Comment:
Thinking about the REST guidelines, I think it would make more sense to use query parameters if you look for books of a specific author. To me, your path is a bit confusing because we're kind of looking through two collections (/books and /authors) at the same time. But it's rather a gut feeling - maybe a code coach can clarify? 😁
I would rather expect for example:
/books/:bookID/authors --> to get the authors of a specific book (if it has several authors)
/authors/:authorname --> if your API didn't have books but authors to get a single author